openapi.yml

```diff
@@ -443,7 +443,7 @@ components:
           nullable: true
           format: int32
           description: "QoS Egress Queue ID"
-          minimum: 0
+          minimum: -1
```

Comment:
Since `queue_id` isn't mandatory, essentially the caller would have two ways to set a default, either not sending `queue_id` or setting `queue_id: -1`, which opens a bit of ambiguity, which is reasonable since it also doesn't break backwards compatibility, but could we get away with only allowing to set a default when `queue_id` isn't set and then not changing the API here with `-1`, and then only using `-1` in the runtime `EVC` but never actually storing it in the DB, and [adapt the `queue_id` projection](https://github.com/kytos-ng/mef_eline/blob/master/db/models.py#L164) allowing it to be undefined (not included)? Just so it follows a same pattern that we have for `spf_attribute`, and then we can keep this pattern if any other attribute also needs a dynamic default? 

I wouldn't reject this PR as it is, I think it's reasonable, but following the existing pattern would be great, I'll leave it up to you and @italovalcy to reach a consensus what you both prefer since you've been already aligning/discussing the requirements, great stuff, @Alopalao.

Reply:
Talked to Italo and decided to go with `-1`. But for completeness, there is a limitation if we do not want an extra value. The updates are different from UI and API:
- The UI sends a completed dictionary of an EVC. If a field is not present in it, we can assure that this non-existent value should be removed. This way we make a difference between non-existent, `None`, and integer.
- The API request presents the possibility to have partial updates. We cannot assure the desire to delete a field if it is not in the dictionary from the request. In this case, we need an extra value to delete `queue_id`.
We can differentiate two types of updates (complete, partial) but since this PR is not that serious I think it is not necessary.
If we have another case where non-existent and `None` should be different, this complication will appear again.

Reply:
Exactly, Aldo. 

Let's go with `-1` then since you guys are on the same page.

Before we close this discussion, can you also document the `-1` in the openapi.yml description, and set it as the default value in the spec too?

Reply:
Added comments.

Reply:
Added comments.